### PR TITLE
Fix prefixed string handling

### PR DESCRIPTION
**Pull request by mcous**
So I made some changes to the grammar file to better support highlighting for s/f/raw/whatever prefixed strings (#10 and #32). I tried to line up the class names with the ones used for similar interpolations in other languages from official atom repos. I've been using this for a while and it works well for me.

Than being said, I'm not sure how compatible it is with different highlighting syntaxes (some cursory perusing of the different syntaxes seem to show a lot of differences). I use solarized-dark, but feedback on other schemes would be good and might necessitate changes.

Here's what it looks like on my machine:
screenshot 2015-08-21 18 34 42
screenshot 2015-08-21 18 34 54

### CODE REVIEW

**Comment by laughedelic** [comment]
Cool :+1: 
Please, let's merge it! Who's responsible for this repo, btw?

**Comment by gangstead** [comment]
Since it got transferred to atom-community I no longer have write access.

**Comment by mcous** [comment]
Per @benogle in #28, @gangstead and @cjwilburn were given write access. At least one of those didn't seem to go through, so anyone at atom-community know what's going on?

**Comment by analytically** [comment]
:thumbsup:

**Comment by benogle** [comment]
Sorry for the delay. I think i've fixed the team permissions. @gangstead you should have write access now.

**Comment by cjwilburn** [comment]
Thanks. It looks like I have permissions now.

**Comment by benogle** [comment]
:+1: @cjwilburn Sorry about that. Oversight on my part

**Comment by gangstead** [comment]
NPM says I don't have permission to do a patch publish.  @benogle are there further github permissions that need to be changed?  NPM still thinks I'm the owner of the package https://www.npmjs.com/package/language-scala

**Comment by gangstead** [comment]
```
➜  language-scala git:(master) ✗ npm access ls-packages                                                                                                                git:(master|…
{
  "comder": "read-write",
  "language-scala": "read-write"
}
➜  language-scala git:(master) npm publish patch                                                                                                                       git:(master|)
npm ERR! publish Failed PUT 403
npm ERR! Darwin 15.0.0
npm ERR! argv "/usr/local/Cellar/node/4.1.1/bin/node" "/usr/local/bin/npm" "publish" "patch"
npm ERR! node v4.1.1
npm ERR! npm  v2.14.4
npm ERR! code E403

npm ERR! You do not have permission to publish 'patch'. Are you logged in as the correct user? : patch
npm ERR!
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/gangstead/code/language-scala/npm-debug.log
```

**Comment by mcous** [comment]
@gangstead perhaps you meant to do `$ apm publish patch`?

**Comment by gangstead** [comment]
Well this is embarrassing.  `v1.1.4` published.

**Comment by gangstead** [comment]
There is also an npm module for some reason and I'm not able to publish to it.  I don't know what that's about.

**Comment by mcous** [comment]
Looks like that was from #9. I think npm publish and apm publish have slightly different semantics. Does `$ npm publish` by itself work?

**Comment by gangstead** [comment]
`npm publish` isn't working for this or my other npm module.  I'm on a new laptop since last publish, but it's not complaining about my user name.  I'll keep digging.

**Comment by gangstead** [comment]
It looks like npm is confused about user names:
```➜  language-scala git:(master) npm whoami                                                                                                                              git:(master|)
gangstead
➜  language-scala git:(master) npm owner ls language-scala                                                                                                             git:(master|)
gangstead <steven@gangstead.com>
```

**Comment by gangstead** [comment]
Basically I can't RTFM.  For npm it's two commands `npm version patch` then `npm publish`.  `npm publish` by itself was failing for lack of a version change.  

I was looking at https://www.npmjs.com/package/npm-publish not noticing that it was a package and not an alias used by `npm publish patch` (in the same way `git add` is an alias for `git-add`).  Without the package installed `npm publish patch` was trying to publish the `patch` [package](https://www.npmjs.com/package/patch), which I don't have access to.

TL;DR - After a brief refresher course both the Atom and NPM packages have the latest language-scala.
